src/components/Prank/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from "react";
+import Cookies from "js-cookie";
+import { Button } from "components/Button";
+import { Modal } from "components/Modal";
+import { StyledLink } from "components/styles";
+import { useToastContext } from "context/toast";
+
+const START_PRANK = new Date("April 1, 2021 11:00:00");
+const END_PRANK = new Date("April 2, 2021 00:00:00");
+const COOKIE_NAME = "PRANK_EXPERIENCED";
+
+export const Prank = () => {
+  const [isVisibleModal, setIsVisibleModal] = useState(false);
+  const { info } = useToastContext();
```

Comment:
nit. I think this is one of the few cases i prefer not to use destructuring `info` is such a generic term. `dispatchToast.info` makes it so much clearer as to the intent of the function.